### DISCUSSION TITLE
Sync from template workflow error

### DISCUSSION THREAD

**Discussion by riasat-sheikh** [alshedivat/al-folio#2751] [Q&A]
I’ve been meaning to ask for help for the last few months! I always get this error when the sync from template workflow runs in my repo. Last time, I manually updated my website, but I would appreciate it if someone could help me fix this.

I suspect this might have something to do with my repo being private? I found something about [PATs](https://github.com/AndreasAugustin/actions-template-sync?tab=readme-ov-file#3-using-a-pat) in the actions-template-sync repo.

<img width="1538" alt="image" src="https://github.com/user-attachments/assets/aca8d5d1-f1cf-4a88-926e-59ca03d4cdbf">

**Comment by george-gca** [comment]
I think I will remove the Sync from template workflow. Every time I tried it I had to end up solving manually.
